nautobot/docs/development/jobs/index.md

```diff
@@ -440,6 +440,9 @@ Similar to `ObjectVar`, but allows for the selection of multiple objects.
 
 An uploaded file. Note that uploaded files are present in memory only for the duration of the job's execution: They will not be automatically saved for future use. The job is responsible for writing file contents to disk where necessary.
 
+!!! note
+    Keep in mind that the maximum size is defined by the FILE_UPLOAD_MAX_MEMORY_SIZE from Django settings. By default, it uses 2621440 (i.e. 2.5 MB).
```

Comment:
Just to be clear - are you saying that files over 2.5MB cannot be uploaded as Job FileVars by default?

Reply:
we found it uploading a big JSON file that fails to serialize when is above this limit

Reply:
Huh. But changing this limit made it work? If I'm reading the Django docs correctly, this is just the threshold at which uploads switch from being stored in-memory to being stored on-disk in a tempdir. It doesn't sound like a cutoff point for max uploads overall. I wonder if we have a bug here.

Reply:
When we increased, it seems we got some weird behavior with the workers... I will double-check before moving this forward

Reply:
@chadell Were you able to get anywhere with this investigation?

Reply:
sorry @glennmatthews , I got distracted. I will try to double-check ASAP

Reply:
@chadell any update on this?

Reply:
no, I haven't tested yet